mobile/src/libs/components/player/components/progress-bar/progress-bar.tsx

```diff
@@ -1,27 +1,37 @@
 import Slider from '@react-native-community/slider';
 import React from 'react';
+import { useColorScheme } from 'react-native';
 
 import { Text, View } from '#libs/components/components';
 import { AppColor } from '#libs/enums/enums';
 import { getFormattedTime } from '#libs/helpers/helpers';
 import { usePlayerControls, usePlayerProgress } from '#libs/hooks/hooks';
 
 import { TRACK_START_TIME } from './libs/constants';
-import { styles } from './styles';
+import { darkStyles, styles } from './styles';
 
 type Properties = {
   isPlaying: boolean;
 };
 
 const ProgressBar: React.FC<Properties> = ({ isPlaying }) => {
+  const colorScheme = useColorScheme();
   const [timeProgress, duration] = usePlayerProgress();
 
   const { handleSeek } = usePlayerControls({ isPlaying });
 
   return (
     <View style={styles.wrapper}>
-      <Text style={styles.progress}>{getFormattedTime(timeProgress)}</Text>
-      <Text style={styles.duration}>{getFormattedTime(duration)}</Text>
+      <Text
+        style={[styles.progress, colorScheme === 'dark' && darkStyles.progress]}
```

Comment:
Can we ever have 'dark' here? You enforced the light mode in styles.xml file

Reply:
@fshabanov okay, i can delete it, but as I said i dont have android device, so i couldnt check how this worked